### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AmadeusGraves**
added the translation section

### CODE REVIEW

**Review by Jayzzzzu — CHANGES_REQUESTED** [review]
Chill

**Comment by AmadeusGraves** [comment]
Hahahahaha sorry, I'm so used to projects with short deadlines that I already have the need to have everything yesterday.

File: README.md [hunk 1]
```diff
@@ -192,6 +195,18 @@ The documentation is being refined and you can contact @JohnGuan via [Discord](h
 
 Help make our documentation better by writing new content for the CasaOS Wiki, correcting existing material, or translating content into new languages.
 
+### Translate CasaOS
```
Comment (JohnGuan) [inline README.md:198]: This part should be added to this paragraph

> ### Helping with Translations 🌍 (WIP)

**Review by JohnGuan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AmadeusGraves) [reply README.md]: Changed

**Comment by AmadeusGraves** [comment]
@JohnGuan John, i updated the readme.md as you wanted, check it when you have time. Cheers!

**Comment by AmadeusGraves** [comment]
@jerrykuku @JohnGuan translation url changed to crowdin.

**Pull request closed without merging**